tests/test_documents.py

```diff
@@ -84,6 +84,12 @@ def test_file_is_zip(self):
         self.assertTrue(file_is_zip(mock_file('file.zip', 1)))
         self.assertFalse(file_is_zip(mock_file('file.sit', 1)))
 
+    def test_file_is_image(self):
+        self.assertTrue(file_is_image(mock_file('file.jpg', 1)))
+        self.assertTrue(file_is_image(mock_file('file.jpeg', 1)))
+        self.assertTrue(file_is_image(mock_file('file.png', 1)))
+        self.assertFalse(file_is_image(mock_file('file.pdf', 1)))
+
```

Comment:
Would be kinda good to add to these some tests with multiple `.`s in. Loads of my files have them, and it's the most basic "your file extension splitter is stupid" check I can imagine.